cmake/herbert_FindMatlab.cmake

```diff
@@ -17,26 +17,63 @@ the path to the directory containing Matlab's dynamically linked libraries
 ``Matlab_UT_LIBRARY``
 the path to the Matlab UT library
 
-``Matlab_VERSION``
+``Matlab_RELEASE``
 the Matlab release e.g. R2019b
 
 See the FindMatlab.cmake documentation for other variables defined by this
 module. You'll find the file bundled with your CMake installation.
 
 #]=======================================================================]
-find_package(Matlab REQUIRED COMPONENTS MAIN_PROGRAM MEX_COMPILER)
+include(MatlabHelpers)
+
+root_dir_changed(_root_changed)
+release_changed(_release_changed)
+if(_root_changed AND NOT _release_changed)
+    # If Matlab_ROOT_DIR has been changed but not Matlab_RELEASE, discard the
```

Comment:
"If Matlab_ROOT_DIR has been changed but not Matlab_RELEASE" is exactly the same as `(_root_changed AND NOT _release_changed)`.

The comment should be why you're discarding the `_RELEASE` in favour of the `_ROOT` (and similarly in the `elseif()`